torch/utils/tensorboard/summary.py

```diff
@@ -589,7 +589,7 @@ def compute_curve(labels, predictions, num_thresholds=None, weights=None):
 
     # Compute bins of true positives and false positives.
     bucket_indices = np.int32(np.floor(predictions * (num_thresholds - 1)))
-    float_labels = labels.astype(np.float)
+    float_labels = labels.astype(np.float_)
```

Comment:
ah yes, missed one here - `float64` is nicer.

Reply:
Oops, my bad :)